src/pages/index.js

```diff
@@ -33,10 +46,47 @@ constants.profileEditButton.addEventListener("click", () => {
   profileFormValidator.resetValidation();
 });
 
-function handleProfileFormSubmit({ name, description }) {
-  user.setUserInfo({ name, description });
+function handleProfileFormSubmit({ name, about }) {
+  profileEditModal.setLoading(true);
+  api
+    .updateUserProfile({ name, about })
+    .then(() => {
+      user.setUserInfo({ name, about });
+    })
```

Comment:
Needs correction: don't forget to handle the api errors  (fix that here and in all the other places)